cvat-ui/src/components/header/settings-modal/styles.scss

```diff
@@ -2,15 +2,15 @@
 //
 // SPDX-License-Identifier: MIT
 
-@import '../../base.scss';
+@import '../../../base.scss';
 
-.cvat-settings-page {
+.cvat-settings-tabs {
     height: 100%;
     overflow-y: auto;
     padding-bottom: 15px;
 
     > div:nth-child(1) {
-        margin-top: 30px;
+        margin-top: 10px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5722655)